priyasirohi09/joplin
# Add dark mode toggle switch to the home screen (#56)

### Summary

This PR addresses issue #56 by adding a dark mode toggle switch directly on the home screen. This enhancement allows users to easily switch between light and dark modes without navigating to the settings menu.

### Changes
- Added a dark mode toggle switch to the home screen.
- Updated the home screen layout to accommodate the new toggle switch.
- Implemented necessary logic to switch themes instantly when the toggle is clicked.

### Screenshots
<img width="1470" alt="Screenshot 2024-08-03 at 20 32 08" src="https://github.com/user-attachments/assets/425eefb4-4dc1-47ac-9f32-72838b8e639a">
<img width="1470" alt="Screenshot 2024-08-03 at 20 32 20" src="https://github.com/user-attachments/assets/91c01923-4e6c-49bb-94c0-1a7de5ae95ac">


### Testing Instructions
1. Open the application.
2. Verify that the dark mode toggle switch is present on the home screen.
3. Click the toggle switch to switch between light and dark modes.
4. Ensure that the theme changes instantly and persists across app restarts.

### Additional Information
- This enhancement improves user experience by providing a quick and accessible way to switch themes.
- No changes were made to the underlying theme implementation; only the toggle switch was added for convenience.

### Related Issues
- Closes #56

### Reviewer Notes
- Please verify the toggle switch functionality on different devices and screen sizes.
- Check for any potential conflicts with existing UI elements on the home screen.
